### ISSUE TITLE
Find in page does not highlight nor jump to the searched word

### ISSUE THREAD

**Issue by btlechowski** [brave/browser-laptop#13943]
### Steps to Reproduce
  1. Restart Brave (Important)
  2. Open about:welcome
  3. Open Find on page
  4. Search 'extensions'
  5. Go forward with the search till you jump back to the beginning

**Actual result:**
The searched words are not highlighted
Find in page does not jump to the searched word
![image](https://user-images.githubusercontent.com/34715963/39274338-f153d288-48e1-11e8-9bf3-249868550eda.png)

After pressing ESC or resizing the window: 
-the highlighted words are shown
-cannot scroll 
-find in page does not jump to the searched word

**Expected result:**
The searched words are highlighted
Find in page jumps to the searched word
![image](https://user-images.githubusercontent.com/34715963/39274452-4f3b70e0-48e2-11e8-80fa-f49adcc8bc3d.png)

**Reproduces how often:**
100%

### Brave Version
```
Brave: 0.22.109 
V8: 6.5.254.41 
rev: 5e4846eb66981a0b7b283ddf7d65a2721641f87d 
Muon: 6.0.3 
OS Release: 6.1.7601 
Update Channel: Beta 
OS Architecture: x64 
OS Platform: Microsoft Windows 
Node.js: 7.9.0 
Brave Sync: v1.4.2 
libchromiumcontent: 65.0.3325.181

```
Also reproducible on Linux

### Additional Information
Related to #13732 #13931

cc @petemill @bsclifton @LaurenWags @kjozwiak @srirambv

**Comment by LaurenWags** [comment]
Could not reproduce on macOS (even using a fresh profile)

**Comment by diracdeltas** [comment]
I have seen this occasionally in 0.22.x before c66 on macos

**Comment by diracdeltas** [comment]
NVM, the bug I saw was where it would show 0 matches even though there were definitely some matches. The screenshot above shows that it correctly identifies the number of matches.

**Comment by petemill** [comment]
Need to wait for the next build (0.22.701) to check on Windows

**Comment by bsclifton** [comment]
Verified this works great on Windows 10 x64 with latest build (0.22.702) 👍

**Comment by GeetaSarvadnya** [comment]
Verified on Windows x64 
* 0.22.702 e4a853d* libchromiumcontent 66.0.3359.139
* muon: 6.0.7

**Comment by kjozwiak** [comment]
@diracdeltas the match case issue was fixed in #14019
